source/_integrations/wilight.markdown

```diff
@@ -25,23 +28,59 @@ The `wilight` integration is to integrate [WiLight](http://www.wilight.com.br) d
 There is currently support for the following device types within Home Assistant:
 
 - Cover (WiLight model C-103).
-- Fan (WiLight model V-104).
-- Light (WiLight model I-100, I-102 and I-107).
+- [Fan](#fan) (WiLight model V-104).
+- Light (WiLight model I-100, I-102, I-107, I-110 and I-112).
+- [Irrigation](#irrigation) / Switch (WiLight model R-105).
 
 {% include integrations/config_flow.md %}
 
 ## Fan
 
-The `wilight` integration allows you to control your Fans from within Home Assistant.
+The `wilight` `fan` integration allows you to control your Fans from within Home Assistant.
```

Comment:
The integration is wilight. The integration has a fan platform. But I don't think we need to specify that to users. It's enough to say that it allows users to control their wilight fan.

Reply:
```suggestion
The `wilight` integration allows you to control your Fans from within Home Assistant.
```